### ISSUE TITLE
Show the next twt event at the front page

### ISSUE THREAD

**Issue by pbruvoll** [pbruvoll/tekkendocs#96]

**Comment by framitdavid** [comment]
I find this issue difficult to implement due to lack of description. What is the best source to fetch this events? :)

**Comment by pbruvoll** [comment]
Yes, this is lacking a description because it was just an idea that I'm not sure how useful it actually will be. So I've added an "Idea" tag
We would need to manually add the events from https://tekken-esports.bn-ent.net/schedule
or https://www.reddit.com/r/Tekken/comments/1bg1orz/all_of_the_current_challenger_master_and_master/
Then we would need some logic that looks at todays date and shows info about the appropriate tournament.

**Comment by framitdavid** [comment]
> Yes, this is lacking a description because it was just an idea that I'm not sure how useful it actually will be. So I've added an "Idea" tag We would need to manually add the events from https://tekken-esports.bn-ent.net/schedule or https://www.reddit.com/r/Tekken/comments/1bg1orz/all_of_the_current_challenger_master_and_master/ Then we would need some logic that looks at todays date and shows info about the appropriate tournament.

Nice to have label like "idea" or "analysis" to figure out if the users needs the feature or not.  But your comment gave me a new idea.

**Suggestion**: 
How about creating a simple forum or topic list where visitors can submit feature requests and vote on them? 🤔

Implementation: We could require users to log in with Discord using OAuth2 (refer to the [Discord OAuth2 Documentation](https://discord.com/developers/docs/topics/oauth2) for details).

Rationale: This feature would help us stay user-centric by prioritizing the features that receive the most votes from the community.

**Comment by pbruvoll** [comment]
Saw the feature request for this, so added some thoughts there.
If we require the user to log in, I think using Discord or Google might be good since I think a lot of players will already have accounts there.
